src/gcc.ts

```diff
@@ -67,11 +69,13 @@ export function distributionUrl(version: string, platform: string): string {
   let archOs: string
   let ext: string
   let winExtraExt = ''
+  let macExtraOS = ''
   switch (platform) {
     case 'darwin':
       osName = 'mac'
       archOs = 'mac'
       ext = 'tar.bz2'
+      macExtraOS = '-10.14.6'
```

Comment:
The macOS download now contains the OS version (I assume used to build GCC) in the filename, so  `${MAC_EXTRA_OS}` has been added to the URL to be replaced.